### PR TITLE
Tree max

### PR DESCRIPTION
**Pull request by ryanapodaca**
- [x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
       - [ ] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [ ] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)

### CODE REVIEW

File: java/datastructures/lib/src/main/java/datastructures/linkedlist/BinaryTree.java [hunk 1]
```diff
@@ -63,4 +63,68 @@ public Object[] postOrder(TreeNode<T> node){
...
+      }
+    }
+  }
+
+  public Integer findMaxFromBinaryTree(){
+    if (root == null){
+      return null;
+    }
+
+    Queue<TreeNode> breadth = new Queue();
```
Comment (lucy-gelderloos) [inline java/datastructures/lib/src/main/java/datastructures/linkedlist/BinaryTree.java:102]: You can nest parameters - try writing this line as:

`Queue<TreeNode<T>> breadth = new Queue<>();`
